packages/Button/src/Button.js

```diff
@@ -45,7 +43,7 @@ const propTypes = {
   isSubmit: PropTypes.bool,
 
   /** The visual style of the button. */
-  kind: PropTypes.oneOf(Kinds.ALL),
+  kind: PropTypes.oneOf(types.ALL),
```

Comment:
does this render correctly on the readme file, types.ALL is a microabstractio maybe would worth to get ride off